website_sale_google_shopping/models/product.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# License, author and contributors information in:
+# __openerp__.py file at the root folder of this module.
+from openerp import models, fields
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    google_product_category_ids = fields.Many2many(
+        string='Categories',
+        comodel_name='google_product_category',
+        relation='product_tmpl2google_product_category',
+        column1='product_tmpl_id',
+        column2='google_product_category_id')
+
+    google_color_id = fields.Many2one(
+        comodel_name='product.attribute',
+        string='Color attribute',
+        help='Fill this field only for apparel items with color attribute.')
+
+    google_size_id = fields.Many2one(
+        comodel_name='product.attribute',
+        string='Size attribute',
+        help='Fill this field only for apparel items with size attribute.')
+
+    google_material_id = fields.Many2one(
+        comodel_name='product.attribute',
+        string='Material attribute',
+        help='Fill this field only for apparel items with material attribute.')
+
+    google_pattern_id = fields.Many2one(
+        comodel_name='product.attribute',
+        string='Pattern attribute',
+        help='Fill this field only for apparel items with pattern attribute.')
+
+    google_condition = fields.Selection(
```

Comment:
Along the same lines of the `product.attribute`, these selections could instead be three records inserted into the `product.attribute.line` w/ a `product.attribute` name of `Condition` or similar.